systems-management/nisysmgmt.yml

```diff
@@ -734,11 +754,21 @@ definitions:
                 value: false
       groups:
         description: Specifies the groups to which the system is assigned.
-        $ref: '#/definitions/ManagedSystemDataProperty'
-        example:
-          lastUpdatedTimestamp: '2019-02-19T11:42:25.078Z'
+        type: object
+        properties:
+          lastUpdatedTimestamp:
+            description: Last updated timestamp.
+            type: string
+            format: iso-date-time
+            example: '2019-02-19T11:42:25.078Z'
           data:
-            - 121b471c-d7d4-4996-a38f-27f7e58be070
+            description: A list of strings defining metadata information about a system.
+            type: array
+            items:
+              type: string
+              example: 
+                - Lab1
```

Comment:
Same comment on whether you intended this example to be an array inside an array.

Reply:
Oops.. copy-paste-wrong-identation case. I fixed it by inlining the example under the 'items', as opposed to un-indent it as it was.